src/npmPackageData/storePackageData.mjs

```diff
@@ -18,12 +20,27 @@ const storePackageData = async () => {
 
   console.log('Retrieved', listOfNpmPackages.length, 'packages names from blob store.');
 
+  // Docs: https://github.com/npm/registry/blob/main/docs/REGISTRY-API.md
   const npmResponses = await Promise.all(listOfNpmPackages.map((packageName) => (
     fetch(`${NPM_REGISTRY_HOSTNAME}${packageName}`)
   )));
 
   const npmData = await Promise.all(npmResponses.map((resp) => resp.json()));
-  const strippedNpmData = npmData.map((data) => stripPackageData(data));
+
+  // Docs: https://github.com/npm/registry/blob/main/docs/download-counts.md
+  const statsResponses = await Promise.all(listOfNpmPackages.map((packageName) => (
+    fetch(`${NPM_REGISTRY_API}downloads/point/last-month/${packageName}`)
+  )));
```

Comment:
Fetch the download data from a different API and map that onto the existing npm package data.